src/main/java/frc/robot/commands/grabber/VisionAlignmentCommand.java

```diff
@@ -0,0 +1,67 @@
+package frc.robot.commands.grabber;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import frc.robot.subsystems.RollerSubsystem;
+import frc.robot.subsystems.RollerSubsystem.HeldPiece;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+import frc.robot.vision.PhotonWrapper;
+
+/**
+ * Intakes 1 game piece with the roller mech. This command runs the rollers at a set power
+ * until the limit switch is triggered.
+ */
+public class VisionAlignmentCommand extends CommandBase {
+    private final BaseSwerveSubsystem driveSubsystem;
+
+    private double angularPower;
```

Comment:
Also doesn't have to be a state; you only need to store things in fields if you need to reference them across multiple methods or you need the previous value when rerunning the same method.